src/cedalion/imagereco/forward_model.py

```diff
@@ -46,7 +46,7 @@ def from_segmentation(
             "skull": "skull.nii",
             "wm": "wm.nii",
         },
-        landmarks_ras_file: str = "landmarks.mrk.json",
+        landmarks_ras_file: Optional[str] = None,
```

Comment:
When landmarks become optional we should use a similar type hint for the member:
`landmarks : Optional[cdt.LabledPointCloud]`